### PR TITLE
Add CI maintenance scripts

### PR DESCRIPTION
**Pull request by tsalo**
Related to #15.

## Changes proposed in this pull request

- Add CI workflows.
- Run black (can replace with ruff in the future).
- Remove fMRIPrep tests.
- Add tox.ini to configure tests.

### CODE REVIEW

File: .maint/ci/activate.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+if [ -e virtenv/bin/activate ]; then
```
Comment (effigies) [inline .maint/ci/activate.sh:1]: I would strongly advise switching to tox instead. I just moved pybids from these kinds of scripts.

**Comment by tsalo** [comment]
@effigies The tests are failing because of problems with the code rather than the configuration, so I think this is ready for your review.

File: .maint/ci/activate.sh (hunk 1 above)
Comment (tsalo) [reply .maint/ci/activate.sh]: I gave it a shot. Thanks!

File: docs/conf.py
Comment (effigies) [inline docs/conf.py:1]: Is this just `black`?

Comment (tsalo) [reply docs/conf.py:1]: Yeah all the changes in this file came from running black

**Pull request merged**